File: TicketOrder/Models/Order.cs
using System;
using System.Collections.Generic;

namespace TicketOrder.Models
{
    public class Order
    {
        public long Id { get; set; }
        public long UserId { get; set; }
        public List<Ticket> TicketsBooked { get; set; }
        public bool Payment { get; set; }
    }
}

File: TicketOrder/models/order.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

// Define Order collection and schema
let Order = new Schema(
  {
    userId: {
        type: String
    },
    ticketsBooked : {
        type: [{
            name: {
                type: String
            },
            available : {
                type: Boolean
            },
            price : {
                type: Number
            }
        }]
    },
    payment : {
        type: Boolean
    },
  },
  {
    collection: 'orders'
  }
);


module.exports = mongoose.model('Order', Order);

class OrderModel {
    constructor(userId, ticketsBooked, payment, _id) {
    this._id = _id
      this.userId = userId;
      this.ticketsBooked = ticketsBooked;
      this.payment = payment;
    }
}

module.exports.OrderModel = OrderModel;